common/services/prismic/articles.js

```diff
@@ -361,8 +360,12 @@ function parseContentLink(document: ?PrismicDocument): ?MultiContent {
 
 function parseArticleDoc(document: PrismicDocument): Article {
   const { data } = document;
+  // When we imported data into Prismic from the Wordpress blog some content
+  // needed to have its original publication date displayed. This is no
+  // longer a feature that editors can, but we still want to display the value
```

Comment:
This sentence still feels like it's missing words after "can".

Reply:
Just simplified the comment to better reflect what's going on.